students/JaeKim/lesson03/assignment/test_basic_operations.py

```diff
@@ -0,0 +1,185 @@
+import customers_model as cm
+import pytest
+from customers_model import TestCustomer
+from basic_operations import add_customer, search_customer, delete_customer, \
+                             update_customer_credit, list_active_customers
+
+"""
+Module docstring
+"""
+
+OK_CUSTOMER = {'customer_id': '12345',
+               'name': 'Suzie',
+               'lastname': 'Edgar',
+               'home_address': '123 Browncroft Blvd, Rochester, NY 97235',
+               'phone_number': '234-123-4567',
+               'email_address': 'suzie@hotmail.com',
+               'status': True,
+               'credit_limit': 40,
+               'stage': 'dev'}
+
+
+def setup_test_db():
+    """
+    Drops test_customer and re-creates an empty test_customer db
+    """
+    # Cleans up Test table from previously failed tests
+    teardown_test_db()
+
+    # Create a Test table
+    cm.database.create_tables([
+        cm.TestCustomer
+    ])
+    cm.database.close()
+
+
+def teardown_test_db():
+    """
+    Drops test_customer, used for teardown
+    """
+    cm.database.drop_tables([
+        cm.TestCustomer
+    ])
+    cm.database.close()
```

Comment:
Great job with the setup and teardown methods. If you use pytest fixtures, you don't have to put them in every test, it automatically runs them for you. 
https://docs.pytest.org/en/latest/fixture.html